### PR TITLE
Remove replicas

### PR DESCRIPTION
**Pull request by harshsharma071988**
[BEGONIA-104] Remove replica count from agent to backend payload
--> Removed logic to create replica count
--> Removed replica count from payload

### CODE REVIEW

File: src/pkg/data/consumers/governor/exporter.go [hunk 1]
```diff
@@ -75,13 +75,12 @@ func (g GovernorExporter) getGovernorAPIPayload() openapi.KubernetesTelemetryReq
 			kubernetesWorkloads.Name = workloadAssessment.Workload.Name
 			kubernetesWorkloads.Kind = workloadAssessment.Workload.Kind
 			kubernetesWorkloads.Namespace = nsa.Namespace.Name
-			kubernetesWorkloads.Replicas = workloadAssessment.Workload.Replicas
 
 			for _, pod := range workloadAssessment.Workload.Pods {
 				containerData := openapi.NewContainerWithDefaults()
 				for _, container := range pod.Containers {
 					containerData.Name = container.Name
-					containerData.Id = container.ID
```
Comment (bgubbala) [inline src/pkg/data/consumers/governor/exporter.go:84]: containerId is temporarily set as optional filed in the governor backend. Please manually change the go-client to treat it as a mandatory field.

Comment (harshsharma071988) [reply src/pkg/data/consumers/governor/exporter.go:84]: Auto generated file should not be touched like this, instead let me know once it is mandatory in backend and i will update the go-client using auto generation.

Comment (pgalache) [reply src/pkg/data/consumers/governor/exporter.go:84]: Let's keep it as it is. Additionally keep in mind this is a change in 0.2.1 branch, which was a temporary one until the new architecture was merged in the main one (which already happened).

**Comment by snootan** [comment]
LGTM

**Review by 4everming — APPROVED** [review]
LGTM

**Pull request merged**